common/src/main/java/tachyon/worker/ClientMetrics.java

```diff
@@ -38,56 +38,111 @@ public ClientMetrics() {
     return defaultMetrics;
   }
 
+  /**
+   * Return current metrics as heartbeat data and reset metrics to default.
+   *
+   * @return current metrics (list of longs)
+   */
   public synchronized List<Long> getHeartbeatData() {
     List<Long> ret = mMetrics;
     mMetrics = createDefaultMetrics();
     return ret;
   }
 
+  /**
+   * Increment BLOCKS_READ_LOCAL counter by the amount specified
+   *
+   * @param n: amount to increment
```

Comment:
Remove the **colon** after the parameter name, or the `javadoc` cannot resolve this correctly.